src/components/ItemList/index.js

```diff
@@ -290,6 +290,27 @@ class ItemList extends Component {
               <div style={{ width: "100%", textAlign: "center" }}>
                 <br />
                 <span> Found {this.state.count} results. </span>
+                <button
+                  onClick={() => {
+                    window.scrollTo({ top: 0, left: 0, behavior: "smooth" });
+                  }}
+                  style={{
+                    cursor: "pointer",
+                    display: "block",
+                    zIndex: 20,
+                    position: "fixed",
+                    padding: "1rem 1.2rem",
+                    border: "none",
+                    fontSize: "11px",
+                    bottom: "40px",
+                    right: "40px",
+                    backgroundColor: "grey",
+                    color: "#fff",
+                    textAlign: "center",
+                  }}
+                >
+                  Scroll to top
+                </button>
```

Comment:
👍🏼 This is amazing and should be the only changes coming from this Pull Request !

Reply:
Yeah thats the only changes coming from the pull request.

Reply:
So, you should remove everything except for those lines, that's what I was requesting up there!
But in the pull request, we still have more than 7k line changes!
Feel free to open a new pull request containing only these 22 lines! 🙏🏽